app/src/main/java/com/yeonae/chamelezone/data/source/remote/member/MemberRemoteDataSourceImpl.kt

```diff
@@ -1,7 +1,61 @@
 package com.yeonae.chamelezone.data.source.remote.member
 
-class MemberRemoteDataSourceImpl private constructor(/*api: Api*/) : MemberRemoteDataSource {
-    override fun createMember() {
+import android.content.Context
+import android.util.Log
+import android.widget.Toast
+import com.google.gson.JsonObject
+import com.yeonae.chamelezone.App
+import com.yeonae.chamelezone.network.api.MemberApi
+import com.yeonae.chamelezone.network.api.RetrofitConnection
+import com.yeonae.chamelezone.network.model.MemberResponse
+import retrofit2.Call
+import retrofit2.Callback
+import retrofit2.Response
+
+class MemberRemoteDataSourceImpl private constructor() :
```

Comment:
```suggestion
class MemberRemoteDataSourceImpl private constructor(private val memberService: MemberService) :
```
이렇게 생성자에서 받아서 사용하는 것으로 변경해본다면 좋을 것 같아요

Reply:
수정했습니당